transformation/bq/Makefile

```diff
@@ -0,0 +1,77 @@
+# Programs
+SED ?= sed
+BQ ?= bq --location=$(BQ_REGION)
+GSUTIL ?= gsutil
+
+POST_INTEGRATION_CLEANUP ?= 1
+
+# Deployment variables
+TURF_BQ_LIBRARY ?= $(BQ_BUCKET_PUBLIC)$(BQ_DATASET_TRANSFORMATION)/turf_library.js
+
+.PHONY: ../turf_library.js
+../turf_library.js:
```

Comment:
Will this module cover other non turf functions? In that case this should be rather
TRANSFORMATION_BQ_LIBRARY and turf_library.js -> transformation_library.js.